include/openenclave/host.h

```diff
@@ -142,6 +154,7 @@ typedef struct _oe_enclave_setting
 #ifdef OE_WITH_EXPERIMENTAL_EEID
         oe_eeid_t* eeid;
 #endif
+        const oe_enclave_setting_config_data* context_config_data;
```

Comment:
Looks like a copy-paste issue, should just be:
```suggestion
        const oe_enclave_setting_config_data* config_data;
```